README.md

```diff
@@ -1,27 +1,39 @@
 # ish - a simple shell written in C
 
 ## Installing
-### Debian/Ubuntu
-check the latest release for debs
-### Other Linux
-compile yourself for now
-## compiling
+
+### To install for Debian/Ubuntu based distributions:
+
+To download for Debian/Ubuntu based distributions, simply go to Github [releases](https://github.com/Itai-Nelken/ish/releases) and install the package through dpkg or apt.
```

Comment:
For the link use https://github.com/Itai-Nelken/ish/releases/latest

That will always redirect to the latest release.

Reply:
:+1: